hack/tests/ci-prepare.sh

```diff
@@ -62,11 +62,24 @@ delete() {
   make delete
 }
 
+check_skip_ci() {
+    if [ ${SKIP_CI:-1} -eq 0 ]; then
+      # No need to run CI
+      exit
+    fi
+}
+
 if [[ ! -e /var/run/secrets/kubernetes.io ]]; then
     reset_xtrace
     return
 fi
 
+set +e
+# check the latest commit
+go run hack/skipci/main.go
```

Comment:
you are not passing the commit in. if i run "git diff-tree --no-commit-id --name-only -r " it errors.